scripts/eslint/rules/boolean-trivia.js

```diff
@@ -1,7 +1,7 @@
-import { AST_NODE_TYPES, TSESTree } from "@typescript-eslint/utils";
-import { createRule } from "./utils";
+const { AST_NODE_TYPES, TSESTree } = require("@typescript-eslint/utils");
+const { createRule } = require("./utils");
 
-export = createRule({
+module.exports = createRule({

```

Comment:
These are all now CJS; a future version of ESLint is poised to completely revamp the way that configuration works and will likely enable ESM rules.